CHANGELOG.md

```diff
@@ -1,68 +1,14 @@
 # CHANGELOG
 
-Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+See the [CONTRIBUTING guide](./CONTRIBUTING.md#Changelog) for instructions on how to add changelog entries.
 
-## [Unreleased]
+## [Unreleased 3.0]
```

Comment:
just spotted this (sorry that i didn't see this the first time around):
these brackets are here to turn this into a link (the URL should be defined at the end of the document; at least that's the case in the currently integrated document). i don't see you defining/changing the URL anywhere. same goes for the other such headers which you changed & also the new document. accordingly, these are no longer links.
please fix.

Reply:
I can escape the brackets here. In the other file these are in backticks so I don't believe markdown will attempt to interpret them as links.

Reply:
(though the example from keepachangelog.com uses these unescaped brackets as version headers)

Reply:
the example from keepachangelog.com has the links defined at the very end. the same is the case for the current version of the changelog here in OpenSearch and i think it should be kept this way

Reply:
Oh of course, I had missed that! Our branching strategy makes this very difficult to do. For example, commits on main include a mix of things that are backported to a 2.x release and things that will only go in 3.0. I don't know how to get a clean commit history that correspond to these labels.